FRAuth/FRAuthTests/FRAuthSwiftTests/FRAuth/Model/OAuth2/TokenTests.swift

```diff
@@ -62,4 +62,32 @@ class TokenTests: FRAuthBaseTest {
             
         }
     }
+    
```

Comment:
One last thing about this file; update the copyright header.

Reply:
That's my kryptonite :P

Updated